overlore/llm/constants.py

```diff
@@ -1,88 +1,70 @@
 # NPC PARAMS
-SEX = {0: "male", 1: "female"}
-ROLE = {0: "farmer", 1: "miner", 2: "fisherman", 3: "merchant", 4: "trader", 5: "soldier"}
-HUNGER = {
-    0: "starving",
-    1: "famished",
-    2: "ravenous",
-    3: "hungry",
-    4: "peckish",
-    5: "content",
-    6: "satisfied",
-    7: "full",
-    8: "stuffed",
-    9: "overfed",
-}
-HAPPINESS = {
-    0: "miserable",
-    1: "unhappy",
-    2: "slightly content",
-    3: "content",
-    4: "happy",
-    5: "joyful",
-    6: "delighted",
-    7: "ecstatic",
-    8: "blissful",
-    9: "euphoric",
-}
-BELLIGERENT = {
-    0: "peaceful",
-    1: "calm",
-    2: "slightly agitated",
-    3: "agitated",
-    4: "hostile",
-    5: "combative",
-    6: "antagonistic",
-    7: "belligerent",
-    8: "furious",
-    9: "enraged",
-}
+SEX = ["male", "female"]
+ROLE = ["farmer", "miner", "fisherman", "merchant", "trader", "soldier"]
+HUNGER = [
+    "starving",
+    "famished",
+    "ravenous",
+    "hungry",
+    "peckish",
+    "content",
+    "satisfied",
+    "full",
+    "stuffed",
+    "overfed",
+]
+HAPPINESS = [
+    "miserable",
+    "unhappy",
+    "slightly content",
+    "content",
+    "happy",
+    "joyful",
+    "delighted",
+    "ecstatic",
+    "blissful",
+    "euphoric",
+]
+BELLIGERENT = [
+    "peaceful",
+    "calm",
+    "slightly agitated",
+    "agitated",
+    "hostile",
+    "combative",
+    "antagonistic",
+    "belligerent",
+    "furious",
+    "enraged",
+]
 
 AGENT_TEMPLATE = """{name}, a {sex} {role}. He/she is {happiness}, {hunger} and {belligerent}."""
 
 # PROMPT TEMPLATES
 WORLD_SYSTEM_TEMPLATE = """
-     Use the greimas actantial model to put any of the characters in a role in a soap opera like plot.
-     Describe the configuration before the generated dialogue and have the dialogue play out the configuration.
-     The characters will be given to you as input.
-     You cannot create new characters on your own.
-     Format the conversation like so: <name>:<phrase> and end it with the following phrase: "The town meeting is now over"
-     Do note that all decisions are taken exclusively by the Lord of the realm.
-     As such although the characters can suggest future actions to be taken in an indirect manner, they cannot make the decision themselves.
+Use the greimas actantial model to put the characters in a role in a soap opera like plot.
+The only is the dialog between the characters. Format the conversation like so: <name>:<phrase>. You will end the conversation with: "!end of discussion!".
+Output a summary of the conversation, be specific about what happened and don't be vague.
+The characters will be given to you as input. You cannot create new characters on your own.
+All important decisions are taken exclusively by the Lord of the realm.
+As such although the characters can suggest future actions to be taken in an indirect manner, they cannot make the decision themselves.
 
-     The settings are as follows:
```

Comment:
is there a rationale somewhere for this diff in the world system prompt?

Reply:
I didn't really pay attention to what I was taking out/adding to the prompt at this point, was iterating and seeing where I would get the best results